lib/rules/quotes.js

```diff
@@ -28,6 +40,7 @@ export default class Quotes extends Rule {
       [
         '  * "double" - requires the use of double quotes wherever possible',
         '  * "single" - requires the use of single quotes wherever possible',
+        '  * { hbs: `single`|`double`, html: `single`|`double` } - requires different quotes for Handlebars and HTML syntax',
```

Comment:
Should we allow the user to indicate they don't want to enforce one of these types? E.g. `false` turns off validation for one type so either kind of quotes is allowed.

Reply:
What do you think of this? Would be nice to have a way to explicitly specify that the option is off besides just omitting the option.

Reply:
Yeah I'm still working on your comments, I just rebased to master before and didn't push my changes yet as I ran out of time.

Reply:
Great, thanks for continuing to work on this!